marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings.tsx

```diff
@@ -556,14 +558,15 @@ const AdvancedSettings: React.FC<Props> = (props) => {
             description = "";
           }
           return (
-            <Alert
+            <HCAlert
               className={styles.alert}
-              type="warning"
+              variant="warning"
               showIcon
               key={warning["level"] + index}
-              message={<div className={styles.alertMessage}>{warning["message"]}</div>}
-              description={description}
-            />
+            >
+              <Alert.Heading>{<div className={styles.alertMessage}>{warning["message"]}</div>}</Alert.Heading>
```

Comment:
@iveyMarklogic, I think it is bad to have to sometimes import the Alert component (when we need Alert.Heading) with an HCAlert component that wraps Alert. Can we refactor HCAlert to have it apply Alert.Heading when needed and pass in the Alert.Heading content as an optional prop?

Reply:
@wooldridge  This is a good point. I added a new prop called `heading`, and I refactored the styles and the unit tests. This prop permits you to pass a string or a jsx component (the jsx is because we could put a link inside the header o another element).